app/uk/gov/hmrc/minorentityidentificationfrontend/views/errorViews/cannot_confirm_business.scala.html

```diff
@@ -0,0 +1,81 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.minorentityidentificationfrontend.config.AppConfig
+@import uk.gov.hmrc.minorentityidentificationfrontend.models.PageConfig
+@import uk.gov.hmrc.minorentityidentificationfrontend.views.helpers.TitleHelper.title
+@import uk.gov.hmrc.minorentityidentificationfrontend.views.html.templates.govuk_template
+@import uk.gov.hmrc.minorentityidentificationfrontend.views.html.helpers._
+
+@this(layout: govuk_template,
+    h1: h1,
+    p: p,
+    govUkHeader: GovukHeader,
+    govukButton: GovukButton,
+    formWithCSRF: FormWithCSRF,
+    govukRadios : GovukRadios,
+    govukErrorSummary: GovukErrorSummary
+)
+
+@(pageConfig: PageConfig, formAction: Call, form: Form[_])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@layout(pageConfig, title = messages("cannot-confirm-business.title")) {
+
+    @if(form.errors.nonEmpty) {
+        @govukErrorSummary(ErrorSummary().withFormErrorsAsText(form))
+    }
+
+    @h1(messages("cannot-confirm-business.heading"))
+
+    @p {
+        @messages("cannot-confirm-business.line_1")
+    }
+
+    @formWithCSRF(action = formAction) {
+        @govukRadios(Radios(
+            fieldset = Some(Fieldset(
+                legend = Some(Legend(
+                content = Text(messages("cannot-confirm-business.line_2")),
+                classes = "govuk-fieldset__legend--m",
+                isPageHeading = true
+                    ))
+                )),
+            idPrefix = Some("yes_no"),
+            name = "yes_no",
+            classes = "govuk-radios--inline",
+            items = Seq(
+                RadioItem(
+                content = Text(messages("app.common.yes")),
+                value = Some("yes")
+            ),
+            RadioItem(
+                content = Text(messages("app.common.no")),
+                value = Some("no")
+                )
+            )
+        ).withFormField(form("do-you-want-to-continue")))
```

Comment:
I'm not certain this is doing much. The apply method on a form attempts to create a Field instance for the form value specified in the input parameter. The form CannotConfirmBusinessForm does not support the value "do-you-want-to-continue".

Reply:
Updated it